.travis.yml

```diff
@@ -18,5 +18,20 @@ matrix:
 
 cache: pip
 
+install:
```

Comment:
@jaraco I think we should apply the same trick in cherrypy (cherrypy/cherrypy#1514) as well.